backend/src/apiserver/main.go

```diff
@@ -40,6 +40,12 @@ import (
 	"google.golang.org/grpc/reflection"
 )
 
+const (
+	HasDefaultBucketEnvVar  = "HASDEFAULTBUCKET"
```

Comment:
Should this be HAS_DEFAULT_BUCKET instead? Same below?

Reply:
Done.